src/main/java/lotto/domain/Lotto.java

```diff
@@ -35,13 +35,13 @@ private boolean isDuplicated(List<LottoNo> lottoNumbers) {
 		return new HashSet<>(lottoNumbers).size() != LOTTO_SIZE;
 	}
 
-	public boolean isContain(LottoNo lottoNo) {
+	public boolean contain(LottoNo lottoNo) {
 		return lottoNumbers.contains(lottoNo);
 	}
 
 	public int compare(Lotto lotto) {
 		return (int)this.lottoNumbers.stream()
-			.filter(x -> lotto.isContain(x))
+			.filter(x -> lotto.contain(x))
```

Comment:
```suggestion
			.filter(Lotto::contain)
```

메서드 레퍼런스를 활용하여 가독성을 상승시킬 수 있습니다!

Reply:
앞으론 스트림을 사용할 땐 최대한 메서드 레퍼런스로
가독성을 향상 시키겠습니다! 감사합니다ㅎㅎ